### ISSUE TITLE
IGA

### ISSUE THREAD

**Issue by brentajones** [alltheplaces/alltheplaces#42]
http://www.iga.com/consumer/locator.aspx

**Comment by vlad555** [comment]
We need all IGA locations in all countries?

**Comment by vlad555** [comment]
![image](https://user-images.githubusercontent.com/9920034/33827820-89d0a056-de1e-11e7-8b7d-a7d9c0e82c9d.png)

**Comment by iandees** [comment]
All countries, if possible.

**Comment by vlad555** [comment]
Ok, all countries which have more than one shop?
Can I create separate spyder for every country because of different logic they store information?

**Comment by vlad555** [comment]
Can I use all australian ZIP-codes to search in australian site?
https://www.iga.com.au/stores/#view=storelocator

**Comment by vlad555** [comment]
https://www.iga.com/international.aspx
Maybe it will be better if I'll got list of countries I should to scrype

**Comment by vlad555** [comment]
may I produce spiders for other countries as separate spiders?

**Comment by iandees** [comment]
Yes, please include the country's domain TLD at the end of the file name like McDonald's.

**Comment by iandees** [comment]
This issue hasn't been worked on in a while. I'm removing the assignment so others can work on it. If you're still working on it, please reassign yourself and reference a pull request or branch where you're doing your work.

**Comment by iandees** [comment]
Closing — the exact site named here (iga.com, US locator) is already fully covered by locations/spiders/iga.py (verified live: 589 items, matches NSI against brand_wikidata=Q3146662). Australia is covered separately by iga_au.py. If broader 'all IGA markets' coverage is wanted, please open new issues per uncovered country (per the international list at iga.com/international.aspx) rather than reopening this catch-all.
